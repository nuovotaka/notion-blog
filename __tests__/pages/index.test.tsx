import { render } from '@testing-library/react'
import Index from '../../src/pages/index'

jest.mock('next/router', () => ({
  useRouter() {
    return {
      asPath: '/',
      pathname: '/',
    }
  },
}))

describe('Index', () => {
  it('renders the page unchanged', () => {
    const { container } = render(<Index firstPost={undefined} />)
    expect(container).toMatchSnapshot()
  })
})
