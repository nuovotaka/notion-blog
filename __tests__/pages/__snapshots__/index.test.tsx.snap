// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders the page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="mainContent"
    >
      <div
        class="noContents"
      >
        There are no contents yet
      </div>
      <footer />
    </div>
    <div
      class="subContent"
    >
      <a
        id="Recommended"
      />
      <div
        class="blogPostLink"
      >
        <h3>
          Recommended
        </h3>
        <div
          class="noContents"
        >
          There are no contents yet
        </div>
      </div>
      <div
        class="blogTagLink"
      >
        <h3>
          Categories
        </h3>
        <div
          class="noContents"
        >
          There are no contents yet
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`RenderPage renders the page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <h2>
        About
      </h2>
      <p>
        Mac関連のこと、日常的な事などを書いていきます。
      </p>
      <p>
        最近は病気の事とレザークラフトのことが多いです。
      </p>
      <p>
        時々更新のブログ。
      </p>
      <p>
        dev.NUOVOTAKA
         powered by
         
        <a
          href="https://github.com/otoyo/easy-notion-blog"
          rel="noopener"
          target="_blank"
        >
          otoyo/easy-notion-blog
        </a>
      </p>
    </div>
  </div>
</div>
`;
